06-func.md

```diff
@@ -52,136 +52,16 @@ Let's try running our function.
 Calling our own function is no different from calling any other function:
 
 ~~~ {.python}
-print 'freezing point of water:', fahr_to_kelvin(32)
-print 'boiling point of water:', fahr_to_kelvin(212)
+print('freezing point of water:', fahr_to_kelvin(32))
+print('boiling point of water:', fahr_to_kelvin(212))
 ~~~
 ~~~ {.output}
 freezing point of water: 273.15
-boiling point of water: 273.15
+boiling point of water: 373.15
 ~~~
 
```

Comment:
> I think we should introduce a different bug so that we can still have the discussion of debugging.

I totally agree on this. I just don't know a good bug to introduce right now and for sake of consistence of the lesson the easy solution is drop this part. Since this will not goes live right now I will request to create one issue for this and doesn't block the merge because of it.

> I alway try to keep in mind that whatever the quirks of a language - we are really using it as a vehicle for teaching programming skills - and debugging is one of them

We think in the same way. =)